docs/notebooks/mister_p.ipynb

```diff
@@ -0,0 +1,8615 @@
+{
```

Comment:
<p>can it explain the coding of the data? For example, what is the meaning of <code>Risk_Strata</code> when it's 0 or 1. Similar for <code>Outcome</code> and <code>Treatment</code> (this last one is easier to infer)   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/716/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>looks great!</p>